src/curl_json.c

```diff
@@ -292,6 +287,14 @@ static int cj_cb_string(void *ctx, const unsigned char *val, yajl_len_t len) {
   return cj_cb_number(ctx, (const char *)val, len);
 } /* int cj_cb_string */
 
+static int cj_cb_boolean(void *ctx, int boolVal) {
+  if (boolVal) {
+   return (cj_cb_number (ctx, "1", 1));
```

Comment:
Please remove the braces around the return value, i.e.:

```c
return cj_cb_number (…);
```

We recently unified the code base on this style.